tests/queries/0_stateless/01834_alias_columns_laziness_filimonov.sh

```diff
@@ -13,16 +13,20 @@ insert into aliases_lazyness(x) select * from numbers(40);
 # In very old ClickHouse versions alias column was calculated for every row.
 # If it works this way, the query will take at least 0.1 * 40 = 4 seconds.
 # If the issue does not exist, the query should take slightly more than 0.1 seconds.
-# The exact time is not guaranteed, so we check in a loop that at least once 
+# The exact time is not guaranteed, so we check in a loop that at least once
 # the query will process in less than one second, that proves that the behaviour is not like it was long time ago.
 
 i=0 retries=300
```

Comment:
Wouldn't it be possible to use sleep profile events instead of timing?

Reply:
Indeed, good idea, thanks!